lib/routes/user.js

```diff
@@ -0,0 +1,57 @@
+import express from 'express';
+import UserService from '../services/user.service';
+import UserValidator from '../validators/user.validator';
+
+const router = express.Router();
+const userService = new UserService();
+const userValidator = new UserValidator();
+
+router.param('user_id', (req, res, next, user_id) => {
+    req.user = userService.find(user_id);
+    if (req.user) {
+        return next();
+    }
+
+    res.status(404).json({ message: `User with id ${user_id} not found` });
+});
+
+router.get('/:user_id', (req, res) => {
+    res.json(req.user);
+});
+
+router.get('/', (req, res) => {
+    const { loginSubstring, limit } = req.query;
```

Comment:
I think it's better to add validation for limit parameter - to be positive number

Reply:
Done